src/whispercpp/context.cc

```diff
@@ -316,7 +317,51 @@ float Context::full_get_token_prob(int segment, int token) {
   return whisper_full_get_token_p(ctx, segment, token);
 }
 
+    // typedef struct whisper_token_data {
+    //     whisper_token id;  // token id
+    //     whisper_token tid; // forced timestamp token id
+
+    //     float p;           // probability of the token
+    //     float plog;        // log probability of the token
+    //     float pt;          // probability of the timestamp token
+    //     float ptsum;       // sum of probabilities of all timestamp tokens
+
+    //     // token-level timestamp data
+    //     // do not use if you haven't computed token-level timestamps
+    //     int64_t t0;        // start time of the token
+    //     int64_t t1;        //   end time of the token
+
+    //     float vlen;        // voice length of the token
+    // } whisper_token_data;
+
+
 void ExportContextApi(py::module &m) {
+  py::class_<whisper_token_data>(m, "WhisperTokenData", "Data for the token")
```

Comment:
```suggestion
  // whisper_token_data -> TokenData
  py::class_<whisper_token_data>(m, "TokenData", "Data for the token")
```

Reply:
nit: I think for the import namespace, there is no need to have the `Whisper` prefix

```python

from whispercpp.api import TokenData

import whispercpp as w

w.api.TokenData ...

```